pulley/src/interp.rs

```diff
@@ -472,6 +472,16 @@ impl XRegVal {
         u64::from_le(x)
     }
 
+    pub fn get_f32(&self) -> f32 {
+        let x = unsafe { self.0.u32 };
+        f32::from_bits(u32::from_le(x))
+    }
+
+    pub fn get_f64(&self) -> f64 {
+        let x = unsafe { self.0.u64 };
+        f64::from_bits(u64::from_le(x))
+    }
```

Comment:
This is technically incorrect because floats are stored in `FRegVal`, not `XRegVal`. I see this is added here for the various macro machinations for crossing between Pulley and the host. Could this be dropped coupled with the suggestion above as well?

Reply:
Done.